packages/terra-form-field/src/Field.scss

```diff
@@ -0,0 +1,84 @@
+@import '~terra-mixins';
+@import './cerner-mock-theme';
+
+:local {
```

Comment:
I'm getting this warning on my textarea code review

` text: 'Custom property ignored: not scoped to the top-level :root element `

Should this be updated here?

Reply:
I think this maybe more related to https://github.com/cerner/terra-core/issues/986 than it is this stylesheet in particular.